share/completions/exif.fish

```diff
@@ -0,0 +1,89 @@
+function __get_target_files
+  set param_args (string split ', ' -- (string trim (string replace -fr "=\w+.*" '' -- (exif --help))))
```

Comment:
`string` supports reading input from a pipe, which makes this less inside-out:

```fish
set param_args (exif` --help | string replace -fr "=\w+.*" '' | string trim | string split ', ')
```

(yes, it also makes the `--` unnecessary)

Reply:
Ah, yes, that looks much better! I switched things around in bb483b38 to prefer pipes when possible.